source/_dashboards/tile.markdown

```diff
@@ -49,6 +49,10 @@ icon_tap_action:
   required: false
   description: Action taken on icon card tap. See [action documentation](/dashboards/actions/#tap-action). By default, it will `toggle` the entity (if possible), otherwise, show the "more-info" dialog.
   type: map
+extras:
+  required: true
```

Comment:
```suggestion
  required: false
```
This cant be required as all types do not support it.

Reply:
Yep, good catch 😅